### DISCUSSION TITLE
Bruno's Desktop and CLI Version

### DISCUSSION THREAD

**Discussion by AkkafaOguz** [usebruno/bruno#1279] [General]
Hi, all. Although the version is displays as v1.5.1 at Bruno app, it's shown as 0.14.0 in CLI as a response to command 'bru --version'. In addition to that, I've tried to update it with the command 'scoop update bruno' but it sill displays as 0.14.0. I don't know if they're related or not but I can not use the output format feature which came with the release 1.5.0 (https://github.com/usebruno/bruno/pull/1187). What should I do to use new output format feature? Thanks for your interest.

**Comment by helloanoop** [comment]
@AkkafaOguz Bruno Desktop API and Bruno CLI use different releases and their version numbers need not be always in sync.

To use the latest cli, you just need to reinstall the cli via command line
```bash
npm install -g @usebruno/cli
```

Then if you check your version `bru --version`, it should be `1.3.0`

![image](https://github.com/usebruno/bruno/assets/9350733/6c9b97bd-4e95-459e-b7b8-d2e77f04c5f0)

Let me know if you're still facing issues.

**Reply by AkkafaOguz** [reply]
Hi @helloanoop, thank for your quick response! It helped a lot. Version displays as 1.3.0 and I can use the new format feature which came with the release 1.5.0 (https://github.com/usebruno/bruno/pull/1187).
